py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -36,59 +36,136 @@ def batch_norm(
     input: TRTTensor,
     weight: Optional[Union[torch.Tensor, np.ndarray]],
     bias: Optional[Union[torch.Tensor, np.ndarray]],
-    running_mean: Optional[Union[torch.Tensor, np.ndarray]],
-    running_var: Optional[Union[torch.Tensor, np.ndarray]],
+    running_mean: Union[TRTTensor, Optional[Union[torch.Tensor, np.ndarray]]],
+    running_var: Union[TRTTensor, Optional[Union[torch.Tensor, np.ndarray]]],
```

Comment:
I just noticed that `torch.ops.aten.native_batch_norm.default` and `torch.ops.aten.batch_norm.default` reuse the function. Since they require `running_mean` and `running_var` to be optional, you can put all these types to `Optional[Union[TRTensor, torch.Tensor, np.ndarray]]`

Reply:
Thank you for your comments! I've resolved the issue based on your feedback.